custodia/secrets.py

```diff
@@ -259,6 +259,8 @@ def _int_get_key(self, trail, request, response):
             output = self.root.store.get(key)
             if output is None:
                 raise HTTPError(404)
+            elif len(output) == 0:
+                raise HTTPError(406)
```

Comment:
A container have an empty string for value. Actually, it's a dict like: `{"type": "simple", "value": ""`